mozci/console/commands/push.py

```diff
@@ -611,6 +611,8 @@ def parse_and_log_details(
             continue
         if classifications_set == {"not classified"}:
```

Comment:
Here we want to check if classification_set only includes `"not classified"` and `"new failure not classified"`, so we should do something like:
```python
if len(classifications_set - {"not classified", "new failure not classified"}) == 0:
```
and then you can remove the `elif` you added.